Makefile

```diff
@@ -128,6 +129,10 @@ staticcheck:
 	$(GO) get honnef.co/go/tools/cmd/staticcheck
 	$(STATICCHECK) ./...
 
+unconvert:
+	$(GO) get github.com/mdempsky/unconvert
```

Comment:
Can we move this to `tools/bin`? 
The `go get` command would pollute the `go.mod` file if it's put here.
See how other tools are managed for example.

Reply:
Fixed.